starter-code/pizza.js

```diff
@@ -61,25 +71,98 @@ function renderGlutenFreeCrust() {
 
 function renderButtons() {
   // Iteration 3: add/remove the class "active" of each `<button class="btn">`
+  if (state.pepperonni) {
+    document.querySelector(".btn-pepperonni").classList.add("active");
+  } else if (!state.pepperoni) {
+    document.querySelector(".btn-pepperonni").classList.remove("active");
+  }
+
+  if (state.mushrooms) {
+    document.querySelector(".btn-mushrooms").classList.add("active");
+  } else if (!state.mushrooms) {
+    document.querySelector(".btn-mushrooms").classList.remove("active");
+  }
+
+  if (state.greenPeppers) {
+    document.querySelector(".btn-green-peppers").classList.add("active");
+  } else if (!state.greenPeppers) {
+    document.querySelector(".btn-green-peppers").classList.remove("active");
+  }
+
+  if (document.querySelector(".sauce").classList.contains("sauce-white")) {
+    document.querySelector(".btn-sauce").classList.add("active");
+  } else {
+    document.querySelector(".btn-sauce").classList.remove("active");
+  }
+
+  if (
+    document.querySelector(".crust").classList.contains("crust-gluten-free")
+  ) {
+    document.querySelector(".btn-crust").classList.add("active");
+  } else {
+    document.querySelector(".btn-crust").classList.remove("active");
+  }
 }
 
 function renderPrice() {
```

Comment:
This as well can be automated by using the ingredients list given at the top of the code:
 let text = "";
  for (let element in state) {
    if (state[element]) {
      price += ingredients[element].price;
      text += `<li>$${ingredients[element].price} ${ingredients[element].name}</li>`;
    }
  }
  document.querySelector(" strong").innerText = `$${price}`;
  document.querySelector(".price ul").innerHTML = text;